src/Backend/Modules/Locale/Ajax/SaveTranslation.php

```diff
@@ -34,12 +34,24 @@ public function execute(): void
         }
 
         // get parameters
-        $language = \SpoonFilter::getPostValue('language', array_keys($possibleLanguages), null, 'string');
-        $module = \SpoonFilter::getPostValue('module', BackendModel::getModules(), null, 'string');
-        $name = \SpoonFilter::getPostValue('name', null, null, 'string');
-        $type = \SpoonFilter::getPostValue('type', BackendModel::getContainer()->get('database')->getEnumValues('locale', 'type'), null, 'string');
-        $application = \SpoonFilter::getPostValue('application', ['Backend', 'Frontend'], null, 'string');
-        $value = \SpoonFilter::getPostValue('value', null, null, 'string');
+        $language = $this->getRequest()->request->get('language');
+        if (!array_key_exists($language, $possibleLanguages)) {
+            $language = null;
```

Comment:
@jonasdekeukelaere can you please read the code and what it does instead of being stubborn when I point out that you are introducing a bug